packages/core/src/App/Containers/RealAccountSignup/currency-selector.jsx

```diff
@@ -171,7 +171,7 @@ const CurrencySelector = ({
                                         <React.Fragment>
                                             <RadioButtonGroup
                                                 id='currency'
-                                                className='currency-selector__radio-group'
+                                                className='with-margin currency-selector__radio-group'
```

Comment:
Let's follow BEM pattern for this to avoid cluttering the global namespace
```suggestion
                                                className='currency-selector__radio-group currency-selector__radio-group--with-margin'
```